fig-files/validation-spec/2024-validations.csv

```diff
@@ -1,81 +1,150 @@
-E0400,pricing_init_rate_period.invalid_numeric_format,pricing_init_rate_period,invalid_numeric_format,Error,Field,pricing_init_rate_period,"* When present, 'adjustable rate transaction: initial rate period' must be a whole number.",NA
-E0401,pricing_init_rate_period.invalid_numeric_value,pricing_init_rate_period,invalid_numeric_value,Error,Field,pricing_init_rate_period,"* When present, 'adjustable rate transaction: initial rate period' must be greater than 0.",NA
+E0400,pricing_init_rate_period.invalid_numeric_format,pricing_init_rate_period,invalid_numeric_format,Error,Field,pricing_init_rate_period,"* When present, 'adjustable rate transaction: initial rate period' must be a whole number.
+",NA
+E0401,pricing_init_rate_period.invalid_numeric_value,pricing_init_rate_period,invalid_numeric_value,Error,Field,pricing_init_rate_period,"* When present, 'adjustable rate transaction: initial rate period' must be greater than 0.
+",NA
 E0420,pricing_fixed_rate.invalid_numeric_format,pricing_fixed_rate,invalid_numeric_format,Error,Field, pricing_fixed_rate,"* When present, 'fixed rate: interest rate' must be a numeric value.",NA
 W0420,pricing_fixed_rate.unreasonable_numeric_value,pricing_fixed_rate,unreasonable_numeric_value,Warning,Field,pricing_fixed_rate,"* When present, 'fixed rate: interest rate' should generally be greater than 0.1.",NA
 E0440,pricing_adj_margin.invalid_numeric_format,pricing_adj_margin,invalid_numeric_format,Error,Field,pricing_adj_margin,"* When present, 'adjustable rate transaction: margin' must be a numeric value.",NA
-W0441,pricing_adj_margin.unreasonable_numeric_value,pricing_adj_margin,unreasonable_numeric_value,Warning,Field,pricing_adj_margin,"* When present, 'adjustable rate transaction: margin' should generally be greater than 0.1.",NA
-E0460,pricing_adj_index_name.invalid_enum_value,pricing_adj_index_name,invalid_enum_value,Error,Field,pricing_adj_index_name,"* 'Adjustable rate transaction: index name' must equal 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 977, or 999.",NA
-E0480,pricing_adj_index_name_ff.invalid_text_length,pricing_adj_index_name_ff,invalid_text_length,Error,Field,pricing_adj_index_name_ff,* 'Adjustable rate transaction: index name: other' must **not** exceed 300 characters in length.,NA
+W0441,pricing_adj_margin.unreasonable_numeric_value,pricing_adj_margin,unreasonable_numeric_value,Warning,Field,pricing_adj_margin,"""* When present, 'adjustable rate transaction: margin' should generally be greater than 0.1.""
+",NA
+E0460,pricing_adj_index_name.invalid_enum_value,pricing_adj_index_name,invalid_enum_value,Error,Field,pricing_adj_index_name,"""* 'Adjustable rate transaction: index name' must equal ""
+""1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 977, or 999.""
+",NA
+E0480,pricing_adj_index_name_ff.invalid_text_length,pricing_adj_index_name_ff,invalid_text_length,Error,Field,pricing_adj_index_name_ff,"""* 'Adjustable rate transaction: index name: other' must **not** exceed 300 characters in length.""
+",NA
```

Comment:
These 3 have a triple quote (`"""`) thing going on. Looks the corresponding validation descriptions have wrapping quotes per line, which I _think_ results in a double quote here, plus the extra wrapping quote.

I tried to edit it below, but it's not really diffing as I'd expect in the UI, so may want to edit yourself to be safe.

```suggestion
W0441,pricing_adj_margin.unreasonable_numeric_value,pricing_adj_margin,unreasonable_numeric_value,Warning,Field,pricing_adj_margin,"* When present, 'adjustable rate transaction: margin' should generally be greater than 0.1.",NA
E0460,pricing_adj_index_name.invalid_enum_value,pricing_adj_index_name,invalid_enum_value,Error,Field,pricing_adj_index_name,"* 'Adjustable rate transaction: index name' must equal ""
""1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 977, or 999.",NA
E0480,pricing_adj_index_name_ff.invalid_text_length,pricing_adj_index_name_ff,invalid_text_length,Error,Field,pricing_adj_index_name_ff,"* 'Adjustable rate transaction: index name: other' must **not** exceed 300 characters in length.",NA
```